### ISSUE TITLE
Pre-workshop survey questions

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/deep-learning-intro#82]
I've been trying to come up with a set of questions to ask learners what material from this lesson they know already. I'm planning on using these with the trial run next week. I've tried to cover the whole range of topics we're currently using but might have missed some topics.  My current questions are: 

> 1. Describe your previous machine learning experience (if any).
> 2. Which of the following can you do already? Answer Yes, Maybe or No for each (don't worry if you can't do any of these):
> Decide if a problem might be solved using deep learning.
> Prepare data so that it can be used to train a neural network.
> Apply a pre-trained model to a dataset.
> Build your own multi-layer neural network using a library such as Keras, PyTorch or Tensorflow.
> Add convolutional or pooling layers to a neural network using a library such as Keras, PyTorch or Tensorflow.
> Use a confusion matrix to evaluate a classifier's accuracy.
> Identify when a network is overfitting its training data.

Does anyone have any suggestions for improvements on these?

**Comment by psteinb** [comment]
I made some experiences with this. I basically asked participants 2-3 questions for each of the following concepts:
- self-estimated proficiency in python coding
- self-estimated proficiency in data science methods
- self-estimated proficiency in ML methods

I decided to rather stick to questions that related to problem solving rather than focusing on libraries.
[Here](https://deeplearning540.github.io/extras/design.html#audience) is a list of questions that I asked in a pre-workshop survey. 

Judging from the answers, the last part on ML went the worst, i.e. my questions were too superficial. Only the question on estimating the t-shirt sizes exposed quite heterogenous answers which I was able to validate during the workshop. The other 2 ML questions suggested that my crowd was very proficient in ML, which on average they were not.
